calcite-qml/Calcite/CheckIndicator.qml

```diff
@@ -14,7 +14,7 @@
  *  limitations under the License.
  ******************************************************************************/
 import QtQuick 2.15
-import QtGraphicalEffects 1.12
+import Qt5Compat.GraphicalEffects
```

Comment:
Please log a follow up issue to investigate alternatives, and possibly remove usage before 200.0

Reply:
Done